### PR TITLE
Make scope of begin/end number variable not global

### PR DESCRIPTION
**Pull request by chaspy**
ref: https://github.com/kamontia/qs/issues/115

### CODE REVIEW

**Comment by kamontia** [comment]
@chaspy If you have time, could you write gotest?

**Comment by chaspy** [comment]
@kamontia Please review and merge

File: main_test.go [hunk 1]
```diff
@@ -19,3 +19,15 @@ func Test_needsChangeMessage_returnFalse(t *testing.T) {
...
 	}
 }
+
+func Test_pickupSquashRange(t *testing.T) {
+  begin, end := pickupSquashRange("2")
+  if begin != 2 || end != 0 {
+    t.Errorf("begin: %v\nend: %v", begin, end)
+  }
+
+  begin, end = pickupSquashRange("2..4")
```
Comment (kamontia) [inline main_test.go]: Add the `pickupSquashRange("4..2")` to check that it is enabled whether swapping is.

Comment (chaspy) [reply main_test.go]: Thanks! addressed https://github.com/kamontia/qs/pull/122/commits/f09718f03b9381f1a101ae0e2247233b9a54fd06

**Comment by kamontia** [comment]
@chaspy Thank you for quick work. Let's keep to refactor like this!!!:smile:

**Pull request merged**